packages/gatsby-plugin-firebase/README.md

```diff
@@ -32,9 +31,44 @@ npm install firebase gatsby-plugin-firebase
 
 ## Usage
 
-### 1. Set up environment variables
+### 1. Register Gatsby plugin
 
-This plugin depends on environment variables. One way to do that is using `dotenv` library:
+In `gatsby-config.js`:
+
+```js
+module.exports = {
+  plugins: [
+    ...otherPlugins,
+
+    {
+      resolve: "gatsby-plugin-firebase",
+      options: {
+        features: {
+          auth: true,
+          database: true,
+          firestore: false,
+          storage: false,
+          messaging: false,
+          functions: true,
+          performance: false,
+        },
+        env: {
+            GATSBY_FIREBASE_API_KEY: '<YOUR_FIREBASE_API_KEY>'
+            GATSBY_FIREBASE_AUTH_DOMAIN: '<YOUR_FIREBASE_AUTH_DOMAIN>'
+            GATSBY_FIREBASE_DATABASE_URL: '<YOUR_FIREBASE_DATABASE_URL>'
+            GATSBY_FIREBASE_PROJECT_ID: '<YOUR_FIREBASE_PROJECT_ID>'
+            GATSBY_FIREBASE_STORAGE_BUCKET: '<YOUR_FIREBASE_STORAGE_BUCKET>'
+            GATSBY_FIREBASE_MESSAGING_SENDER_ID: '<YOUR_FIREBASE_MESSAGING_SENDER_ID>'
+            GATSBY_FIREBASE_APP_ID: '<YOUR_FIREBASE_APP_ID>'
+            GATSBY_FIREBASE_MEASUREMENT_ID: '<YOUR_FIREBASE_ANALYTICS_TRACKING_ID>'
+        }
```

Comment:
Let's change this to:

```
{
  resolve: "gatsby-plugin-firebase",
  options: {
    features: {
      auth: true,
      firestore: true,
    },
    credentials: {
       apiKey: "<YOUR_FIREBASE_API_KEY>",
       authDomain: "<YOUR_FIREBASE_AUTH_DOMAIN>",
       databaseURL: "<YOUR_FIREBASE_DATABASE_URL>",
       projectId: "<YOUR_FIREBASE_PROJECT_ID>",
       storageBucket: "<YOUR_FIREBASE_STORAGE_BUCKET>",
       messagingSenderId: "<YOUR_FIREBASE_MESSAGING_SENDER_ID>",
       appId: "<YOUR_FIREBASE_APP_ID>",
       measurementId: "<YOUR_FIREBASE_MEASUREMENT_ID>",
    },
  },
},
```

I like `credentials` better than `env` because that's v0.2 API gonna be, as well as how `gatsby-theme-firebase` does it. Also, let's make it a normal config object instead of the `GATSBY_FIREBASE_API_KEY` thing.